### PR TITLE
Added break line action

### PR DESCRIPTION
**Pull request by AndreasArvidsson**
`"break air"`

## Checklist

- [x] I have added [tests](https://www.cursorless.org/docs/contributing/test-case-recorder/)
- [x] I have updated the [docs](https://github.com/cursorless-dev/cursorless/tree/main/docs) and [cheatsheet](https://github.com/cursorless-dev/cursorless/tree/main/cursorless-talon/src/cheatsheet)
- [x] I have not broken the cheatsheet

### CODE REVIEW

File: packages/cursorless-engine/src/actions/BrakeLine.ts [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  for (const range of contentRanges) {
+    const position = range.start;
+    const line = document.lineAt(position);
+    const indentation = line.text.slice(
+      0,
+      line.firstNonWhitespaceCharacterIndex,
+    );
+
+    edits.push({
+      range: position.toEmptyRange(),
```
Comment (pokey) [inline packages/cursorless-engine/src/actions/BrakeLine.ts:55]: This leaves trailing whitespace on first line, right? I'd argue we should remove it

File: packages/cursorless-engine/src/actions/Actions.ts [hunk 2]
```diff
@@ -1,7 +1,8 @@
-import { TreeSitter } from "../typings/TreeSitter";
 import { Snippets } from "../core/Snippets";
 import { RangeUpdater } from "../core/updateSelections/RangeUpdater";
 import { ModifierStageFactory } from "../processTargets/ModifierStageFactory";
+import { TreeSitter } from "../typings/TreeSitter";
+import BrakeLine from "./BrakeLine";
```
Comment (pokey) [inline packages/cursorless-engine/src/actions/Actions.ts:5]:
```suggestion
import {BreakLine} from "./BreakLine";
```

**Review by pokey — COMMENTED** [review]
Looks good with minor comments. Also please update docs 😊

File: packages/cursorless-engine/src/actions/BrakeLine.ts (hunk 1 above)
Comment (AndreasArvidsson) [reply packages/cursorless-engine/src/actions/BrakeLine.ts]: fixed

**Pull request merged**